### PR TITLE
Fix: Fix paddings and empty item behavior [ED-11840]

### PR DESCRIPTION
**Pull request by murzin-nikita**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by TzviRabinovitch — DISMISSED** [review]

File: assets/dev/scss/frontend/_container.scss [hunk 1]
```diff
@@ -137,8 +137,8 @@
 		width: 100%;
 		max-width: var(--content-width);
 		margin: 0 auto;
-		padding-inline-start: 0;
-		padding-inline-end: 0;
+		padding-inline-start: var(--bc-padding-inline-start, 0);
```
Comment (hein-obox) [inline assets/dev/scss/frontend/_container.scss:140]:
```suggestion
		padding-inline-start: 0;
```

File: assets/dev/scss/frontend/_container.scss [hunk 2]
```diff
@@ -137,8 +137,8 @@
...
+		padding-inline-end: var(--bc-padding-inline-end, 0);
```
Comment (hein-obox) [inline assets/dev/scss/frontend/_container.scss:141]:
```suggestion
		padding-inline-end: 0;
```

File: assets/dev/js/frontend/handlers/container/grid-container.js [hunk 3]
```diff
@@ -56,22 +56,23 @@ export default class GridContainer extends elementorModules.frontend.handlers.Ba
 	}
 
 	initLayoutOverlay() {
+		this.getCorrectContainer();
+		// Re-init empty view element after container layout change
+		const selectors = this.getSettings( 'selectors' ),
+			isGridContainer = this.$element[ 0 ].classList.contains( 'e-grid' );
```
Comment (hein-obox) [inline assets/dev/js/frontend/handlers/container/grid-container.js:62]:
```suggestion
			isGridContainer = this.$element[ 0 ]?.classList.contains( 'e-grid' );
```

Comment (hein-obox) [reply assets/dev/js/frontend/handlers/container/grid-container.js:62]:
```suggestion
			isGridContainer = this.$element.hasClass( 'e-grid' );
```

File: assets/dev/js/frontend/handlers/container/grid-container.js [hunk 4]
```diff
@@ -56,22 +56,23 @@ export default class GridContainer extends elementorModules.frontend.handlers.Ba
...
+		this.elements.emptyView = this.findElement( selectors.emptyView )[ 0 ];
```
Comment (hein-obox) [inline assets/dev/js/frontend/handlers/container/grid-container.js:63]: Maybe

```suggestion
		this.elements?.emptyView = this.findElement( selectors.emptyView )[ 0 ];
```

**Review by hein-obox — DISMISSED** [review]

File: assets/dev/js/frontend/handlers/container/grid-container.js [hunk 5]
```diff
@@ -56,22 +56,24 @@ export default class GridContainer extends elementorModules.frontend.handlers.Ba
 	}
 
 	initLayoutOverlay() {
+		this.getCorrectContainer();
+		// Re-init empty view element after container layout change
+		const selectors = this.getSettings( 'selectors' ),
+			isGridContainer = this.getElementSettings( 'container_type' );
```
Comment (louiswol94) [inline assets/dev/js/frontend/handlers/container/grid-container.js:62]:
```suggestion
			isGridContainer = 'grid' === this.getElementSettings( 'container_type' );
```
?

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**